spec/rake_tasks/abother_dummy_rake_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rake'
+
+describe 'Another dummy rake' do
+  describe "another_dummy:do_something_once" do
+    let(:task_name) { "another_dummy:do_something_once" }
+    let(:task) { Rake::Task[task_name] }
+
+    context 'when Rake.load_rakefile is used to load rake task' do
+      before(:all) do # :all must be used to ensure the rake task is loaded only once
+        Rake.load_rakefile("tasks/another_dummy.rake")
+        Rake::Task.define_task(:environment)
+      end
+
+      after do
+        Rake::Task[task_name].reenable
+
+        # reset the state that was changed by the rake task execution
```

Comment:
recommendation

I'd say it's clear from the code (and examples' description):

```suggestion

        # reset the state that was changed by the rake task execution
```